package.json

```diff
@@ -22,104 +22,103 @@
   },
   "dependencies": {
     "@date-io/luxon": "^2.10.8",
```

Comment:
@jaruesink, curious about his: Why do we include both the `@date-io/luxon` and `luxon` packages in here? Can we just remove the `luxon` dependency, if it is now name-spaced in `@date-io`?